src/table/components/TableBodyWrapper.jsx

```diff
@@ -98,6 +98,8 @@ function TableBodyWrapper({
                       selectionState,
                       cell,
                       selectionDispatch,
+                      tableRows: rows,
```

Comment:
I think this is not a good idea. You are basically putting all the table data on every cell. I see you need qElemNumber for the cell above and bellow. If you have to pass multiple cells, I would prefer only sending those two cells extra cells that are needed

Reply:
I agree that putting all the table data on every cell is not good. It is a bit overkill and would be good to pass one-column data.

But putting all the table data on every cell does not impact performance. It is a copy by reference.

And getting two cells extra cells for every cell impacts performance. As it needs to do operations on the array.

Reply:
what about putting the previous and next cell `qElemNumber` on the cell in `handleData`? That way you don't have to add any arguments at all.